scripts/UpdateDependencyVersions.ps1

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env powershell
+#requires -version 4
+
+<#
+.SYNOPSIS
+The goal of the script is to make it easier for CI to automatically update and make changes to this repository, without needing
+to know the internal details of how config files are layed out in this repo.
+#>
+[cmdletbinding(SupportsShouldProcess = $true)]
+param(
+    [string]$DotNetSdkVersion,
+    [string]$DotNetRuntimeVersion,
+    [switch]$Force
+)
+
+$ErrorActionPreference = 'Stop'
+$git = Get-Command git -ErrorAction Ignore
+
+if (!$git) {
+    Write-Warning 'Missing recommmended command: git'
+}
+
+$updates = @()
+
+if ($DotNetSdkVersion -and $PSCmdlet.ShouldProcess("Update dotnet SDK to $DotNetSdkVersion")) {
```

Comment:
It's a "dry run" powershell convention. Allows users to run `./my_script.ps1 -WhatIf` and see what _would_ happen if the flag had not been specified.